.idea/modules.xml

```diff
@@ -6,6 +6,10 @@
       <module fileurl="file://$PROJECT_DIR$/.idea/core.iml" filepath="$PROJECT_DIR$/.idea/core.iml" />
       <module fileurl="file://$PROJECT_DIR$/.idea/sponge.iml" filepath="$PROJECT_DIR$/.idea/sponge.iml" />
       <module fileurl="file://$PROJECT_DIR$/.idea/taskchain.iml" filepath="$PROJECT_DIR$/.idea/taskchain.iml" />
+      <module fileurl="file://$PROJECT_DIR$/bukkit/taskchain-bukkit.iml" filepath="$PROJECT_DIR$/bukkit/taskchain-bukkit.iml" />
```

Comment:
dupe modules here but already have the .iml files on ignore so i think we just need to drop this file and ignore it.